src/core/timer.css

```diff
@@ -7,4 +7,6 @@
     height: 40px;
     text-align: center;
     font-size: x-large;
+    padding-top: 10px;
```

Comment:
I was testing these changes locally and this padding seems to have added bit weird placing of the timer on few pages. Please find the screenshot below:
![image](https://user-images.githubusercontent.com/10597872/94990348-f5af3400-0598-11eb-85b0-13165ad063cc.png)
I tried it on github.com and leetcode.com and this came out this way. However, when I tried it on google.com this appeared similar to the gif. Can you check this out?

Reply:
Yes

Reply:
I fixed this issue!